indico/modules/events/abstracts/controllers/display.py

```diff
@@ -44,3 +46,29 @@ def _process(self):
         pdf = AbstractToPDF(self.abstract)
         file_name = secure_filename('abstract-{}.pdf'.format(self.abstract.friendly_id), 'abstract.pdf')
         return send_file(file_name, pdf.generate(), 'application/pdf')
+
+
+class RHDisplayReviewableTracks(RHConferenceBaseDisplay):
+    def _process(self):
+        user = session.user
+        tracks_set, events_set = set(), set()
```

Comment:
why initialize them empty first? just create them like this:

    tracks_set = user.convener_for_tracks | user.abstract_reviewer_for_tracks
    events_set = user.global_convener_for_events | user.global_abstract_reviewer_for_event